Source/RapyutaSimulationPlugins/Private/Core/RRSDFParser.cpp

```diff
@@ -587,13 +587,19 @@ bool FRRSDFParser::ParseSensorsProperty(const sdf::ElementPtr& InLinkElement, TA
         {
             auto& lidarInfo = sensorProp.LidarInfo;
 
+            lidarInfo.TopicName = URRCoreUtils::StdToFString(sensorElement->Get<std::string>(SDF_ELEMENT_SENSOR_TOPIC));
+            ensure(false == lidarInfo.TopicName.IsEmpty());
+            lidarInfo.FrameId = URRCoreUtils::StdToFString(sensorElement->Get<std::string>(SDF_ELEMENT_SENSOR_FRAME_ID));
+            lidarInfo.PublicationFrequencyHz = sensorElement->Get<float>(SDF_ELEMENT_SENSOR_UPDATE_RATE);
+            ensure(lidarInfo.PublicationFrequencyHz > 0.f);
```

Comment:
I think we don't need this check. sometimes user want to add lidar whose publication is triggered by user or want to manage loop by themselves